packages/react/src/components/ContainedList/ContainedList.mdx

```diff
@@ -23,6 +23,17 @@ import { Props, Story, Preview } from '@storybook/addon-docs';
   <Story id="components-containedlist--default" />
 </Preview>
 
+### Search
+
+We have added in the capability to pass down the `Search` component within the `ContainedList` action prop.
+The `Search` component itself is not a prop but does render with specific styling such that the `Search` bar appears below the Contained List title as opposed to rendering inline.
+The `Search` component will also remain sticky under the title, so that it remains visible on scroll, when there are many list items passed in. 
+## Render Custom Icon
+
+<Preview>
+  <Story id="components-containedlist--with-search" />
+</Preview>
+
```

Comment:
Could we add some sample code before the preview? either whole or broken into pieces

```
const [searchTerm, setSearchTerm] = useState('');
const [searchResults, setSearchResults] = useState([]);
const handleChange = (event) => {
    setSearchTerm(event.target.value);
  };

useEffect(() => {   
  const listItems = [
      'List item 1',
      'List item 2',
      'List item 3',
      'List item 4',
      'List item 5',
      'List item 6',
      'List item 7',
      'List item 8',
  ];

  const results = listItems.filter((listItem) =>
    listItem.toLowerCase().includes(searchTerm.toLowerCase())
  );
    setSearchResults(results);
}, [searchTerm]);

return (
    <ContainedList
      label="List title"
      kind="on-page"
      action={
        <Search
          placeholder="Search"
          value={searchTerm}
          onChange={handleChange}
          size="lg"
        />
      }>
      {searchResults.map((listItem, key) => (
        <ContainedListItem key={key}>
          {listItem}
        </ContainedListItem>
      ))}
    </ContainedList>
```